library/cwm/examples/date_time.rb

```diff
@@ -19,6 +21,8 @@ def label
 
 class EventDate < CWM::DateField
   def initialize
+    super
+
```

Comment:
NP: Just curious about why in yast2-bootloader you kept `textdomain` before `super` and here is the other way around.

Reply:
well, it does not matter. It is order independent as textdomain is added to domain pool and not set exclusively for given object (it will be hard to do it in ruby).